packages/plugin-ext/src/hosted/node/hosted-plugin-process.ts

```diff
@@ -117,6 +118,9 @@ export class HostedPluginProcess implements ServerPluginRunner {
         env.PATH = process.env.PATH;
         // add HOME to env since some plug-ins need to read files from user's home dir
         env.HOME = process.env.HOME;
+        if (argv.extensionTestsPath) {
```

Comment:
it's not overkill to use bring yargs dependency there ?

Reply:
`yargs` should be always available

but I better to replace it with `CliContribution` that we parse args only once